.github/workflows/pr-tests.yml

```diff
@@ -31,10 +31,12 @@ jobs:
         key: ${{ runner.os }}-pilcom-node-modules
     - name: Install pilcom
       run: git clone https://github.com/0xPolygonHermez/pilcom.git  && cd pilcom && npm install
+    - name: Check without Halo2
+      run: cargo check
```

Comment:
Same q here

Reply:
Because it is built with halo2 later, and build includes check.